src/core/cookie.js

```diff
@@ -124,7 +104,15 @@ DM.provide('Cookie',
     {
         if (session)
         {
-            DM.Cookie.setRaw(DM.QS.encode(session), session.expires, session.base_domain);
+            var key = 'dms_' + DM._apiKey;
+            var value = DM.QS.encode(session);
+
+            var options = {
+              expires: new Date(session.expires * 1000).toGMTString()
+            };
```

Comment:
Shouldn't we use the `domain` options here?
``` javascript
var options = {
  expires: new Date(session.expires * 1000).toGMTString(),
  domain: session.base_domain
};
```
See Cookies.js [documentation](https://github.com/ScottHamper/Cookies#cookiessetkey-value--options)